data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,227 @@ mission "Remnant: Visit to Paradise 1"
+			choice
+				`	"I think I get the picture. Let me know if I can help."`
+					goto shatteredhelp
+				`	"What is this 'Primary Weft' you have mentioned?"`
+			`	Another Remnant steps forward. "The Weft is what we term the mesh of dimensions, energy, and time that form a subset of the universe. A layer, of sorts." He looks thoughtful. "What you probably know as 'Hyperspace' is what we technically call the secondary weft. There are a few others we are fairly confident exist, and several that we have hypothesized might exist but we have no means of confirming. Practically, each of these overlaps with the primary weft to some extent." His gestures take on an amused tone. "Of course, what we call the Primary Weft is only primary to those of us who live here. For all we know, there may be people living elsewhere that consider their own weft to be the primary one. Nothing says ours is special or the best or anything of the sort."`
+			label shatteredhelp
+			`	"In terms of help, could you go to Caelian? We have sent word ahead, and our engineers are retrieving some of our remaining equipment from the time of the Exodus. We are hoping that it might be of use to help us repair the Shattered Light, or at least understand what happened to it.`
+			`	"Before you leave, though, there was some stuff that survived intact we would like to share with you." Behind her five androids emerge from one of the cargo holds, each one carrying a large trunk of equipment. "These are known as NDR-114 Androids, and were cutting edge innovations back when the exodus occurred. Don't expect too much of them: They are not self-aware, sentient, or even particularly advanced artificial intelligences. They are, however, excellent extensions of a ship's self-maintenance systems. As it turns out, a large number of these androids were instrumental in keeping the Shattered Light flying all these years. We have plenty to study, we thought you might appreciate having a few." She makes a dismissive gesture. "If they end up not being helpful for you, our outfitters will be able to re-assign them to another ship."`
+			scene "outfit/android"
+			`	"Embers light your path, Captain <first>." With that, the crowd disperses.`
+		outfit "NDR-114 Android" 5
+
+
+
+mission "Remnant: Shattered Light 5"
+	name "Brilliant Energy"
+	description "Retrieve a team with their equipment from <destination> to help research the Shattered Light."
+	landing
+	source "Viminal"
+	destination "Caelian"
+	to offer
+		has "Remnant: Shattered Light 4: done"
+
+
+
+mission "Remnant: Shattered Light 6"
+	name "Shattered Light"
+	description "Ferry the team and their gear back to <destination>"
+	blocked "From your count at the information desk, you estimate that you need space for 11 people as well as 18 cargo."
+	cargo "equipment" 18
+	passengers 11
+	landing
+	source "Caelian"
+	destination "Viminal"
+	to offer
+		has "Remnant: Shattered Light 5: done"
```

Comment:
```suggestion
		has "Remnant: Shattered Light 5: offered"
```
Or:
```suggestion
		has "Remnant: Shattered Light 4: done"
```

Reply:
No, SL 5 should complete as normal, and be used as a "done" mission to test for here. In the absence of any "to complete" conditions, it will complete merely with the act of arriving at Caelian; after which SL 6 can trigger.

Reply:
It looks like the problem I was having was because of the lack of `accept` on the offer conversation.